### PR TITLE
SMGs now count as handguns

### PR DESCRIPTION
**Pull request by SonixApache**
🆑 
 - tweak: SMGs now count as handguns

### CODE REVIEW

**Comment by RealestEstate** [comment]
Akimbo SMG is going to be dope

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -166,6 +173,8 @@
 	load_method = 2
 	var/cover_open = 0
 
+/obj/item/weapon/gun/projectile/automatic/l6_saw/isHandgun()
```
Comment (DamianX) [inline code/modules/projectiles/guns/projectile/automatic.dm:176]: The description of this thing says it's a LMG but you say you're making SMGs count as handguns.
t. can't into guns

Comment (SonixApache) [reply code/modules/projectiles/guns/projectile/automatic.dm:176]: `return FALSE	`, this gun is unaffected

every automatic is a child of the normal R&D SMG for some god awful reason, but that's the one i'm changing

Comment (DamianX) [reply code/modules/projectiles/guns/projectile/automatic.dm:176]: oh, my bad

**Comment by Intigracy** [comment]
Does this affect anything other than akimbo?

**Comment by DamianX** [comment]
yes, handguns can be holstered

**Pull request merged**